main.cpp

```diff
@@ -53,83 +53,114 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 //  1)
 //  Noun: synth
-//  action 1: synth.playNote();
+//  action 1: The synth plays the note that's pressed on the keyboard.
+synth.playNote();
 
-//  action 2: synth.arpeggiateNotes();
+//  action 2: The synth activates its arpeggiating function.
+synth.arpeggiateNotes();
 
-//  action 3: synth.modulateEffect();
+//  action 3: A synth parameter modulates an effect.
+synth.modulateEffect();

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.